app/Importers/DataImporter.php

```diff
@@ -12,16 +12,22 @@
 use App\Models\ImportInfoDetail;
 use App\Services\MapboxGeocodingService;
 use GuzzleHttp\Client;
+use Stichoza\GoogleTranslate\GoogleTranslate;
 
 abstract class DataImporter
 {
+    public const language = "en";
+
     protected bool $stopExecution = false;
     protected int $importInfoId;
+    protected GoogleTranslate $translate;
 
     public function __construct(
         protected Client $client,
         protected MapboxGeocodingService $mapboxService,
-    ) {}
+    ) {
+        $this->translate = new GoogleTranslate();
```

Comment:
Why not in the constructor?

Reply:
same as above

Reply:
and the reason to having it here was this problem with tests - i gave 3 arguments, but it said that i gave only 2
https://github.com/blumilksoftware/escooters/actions/runs/5887951907/job/15968210359